src/cli/global/mod.rs

```diff
@@ -9,11 +9,9 @@ mod upgrade_all;
 
 #[derive(Debug, Parser)]
 pub enum Command {
-    // BREAK: This should only have the `i` as an alias
-    #[clap(visible_alias = "i", alias = "a")]
```

Comment:
Are you going to add `add`?

Reply:
Seeing this was 3 months ago lets just merge this!

Reply:
Can you approve it then? :)